universe/auto_syncer.go

```diff
@@ -764,8 +764,8 @@ func (f *FederationEnvoy) SyncServers(serverAddrs []ServerAddr) error {
 	return nil
 }
 
-// SetAllowPublicAccess sets the global sync config to allow public access
-// for proof insert and export across all universes.
+// SetAllowPublicAccess sets the global (default) sync config to allow public
+// access for proof insert and export across all universes.
```

Comment:
Sorry, it's been a while since I last looked at this, so I might misunderstand something. But isn't this config only for what _our_ syncer does? And doesn't influence what others can do to our universe? If that's true, then we might want to rename this method as its name first threw me off due to its similarity to the old boolean flag we now replaced.